src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -174,16 +175,24 @@ def return_input_paths(job, work_dir, ids, *args):
 
     return paths.values()
 
+def mock_mode():
```

Comment:
I deliberately put mock_mode next to docker_call for simplicity. I also put an identical function in the überscript because importing between it and the pipelines introduces a bunch of dependencies.